auto_round/autoround.py

```diff
@@ -1319,6 +1319,8 @@ def get_quantized_layer_names_outside_blocks(self):
                 continue
             try:
                 layer = get_module(self.model, key)
+                if layer == self.model:
```

Comment:
For a straightforward approach, consider listing all the supported_layers in self.model.named_module